rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -495,9 +465,12 @@ void RSocketStateMachine::handleConnectionFrame(
           auto streamToken = resumeManager_->getStreamToken(streamId);
           auto subscriber = coldResumeHandler_->handleRequesterResumeStream(
               streamToken, consumerAllowance);
+          //TODO(somatsun): ensure that subscription is called from the correct
```

Comment:
Can you use `EventBaseManager::get()->getCurrentEventBase()` and use it here directly to create `ScheduledSubscriptionSubscriber` instead ?